doc/source/reference/random/brng/threefry32.rst

```diff
@@ -0,0 +1,36 @@
+ThreeFry32 Counter-based RNG
```

Comment:
This generator should probably be dropped. It is only really slightly useful on 32 bit platforms.

Reply:
I'll wait to hear back whether this, and the other two, should be removed.  Probably a good idea to start with a smaller API since easier to expand than contract.  This can always go in a stand along package of bit-generators along with PCG32/64.

Reply:
+1 for removal.